firmware/firmware.md

```diff
@@ -11262,12 +11262,16 @@ make: *** [user] Error 2
 
 When you are using the Particle Cloud to compile your `.ino` source code, a preprocessor comes in to modify the code into C++ requirements before producing the binary file used to flash onto your devices.
 
-However, there might be instances where the preprocessor causes issues in your code. One example is the use of class/structs in your function parameters.
+However, there might be instances where the preprocessor causes issues in your code. Examples include:
 
+  * Using class/struct types in your function parameters (as above).
+  * Using single variables in conditions - e.g. `else if(foo)`
```

Comment:
hmm... interesting. Added an issue here: https://github.com/spark/buildpack-wiring-preprocessor/issues/6

I would prefer to fix the issue than listing as an issue in the preprocessor since that should work